bundles/org.openhab.binding.ism8/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ism8"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="device">
+		<label>Ism8 Binding</label>
+		<description>Ism8 Interface</description>
+
+		<config-description>
+			<parameter name="portNumber" type="integer">
```

Comment:
You might want to add the `required` attribute and min/max values.

Reply:
I have added the required attribute. I'm not so sure about the min. and max. . Basically does this not depend on the ISM8 card, but more on a free port on the device where OpenHab is running. This binding is opening the port and waiting for receiving values from the ISM8. This port needs to be entered in the Web-Front-End of the ISM8. The ISM8 is sending the values normally without any active command.

Reply:
The valid range is defined by the IP protocol. E.g. by specifiying min=1 you don't need to deal, if the user enters a negative value. Max value would be 65535.

Reply:
OK, added